bot/exts/help_channels/_caches.py

```diff
@@ -20,3 +20,7 @@
 # This cache maps a help channel to original question message in same channel.
 # RedisCache[discord.TextChannel.id, discord.Message.id]
 question_messages = RedisCache(namespace="HelpChannels.question_messages")
+
+# This cache keeps track of the dynamic message ID for
+# the continuously updated message in the #How-to-get-help channel.
+dynamic_message = RedisCache(namespace="HelpChannels.dynamic_message")
```

Comment:
I think redis is good enough, but a more robust approach would be to use message history to find the message when the cog is initialised. It would only need to get the most recent message and make sure the author is the bot (and optionally check the start of the message is "Currently available help channel").

Reply:
We can stick with the Redis cache for now since I think we'll probably want to shove more information in the dynamic message in the future.